content/en/service_management/workflows/private_actions/_index.md

```diff
@@ -0,0 +1,195 @@
+
+   You can safely ignore the error `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
+
+
+{{% /tab %}}
+
+{{% tab "Docker Compose" %}}
+1. Click **Docker Compose**.
+1. Create a `docker-compose.yaml` file and add the provided YAML, or add the `runner` stanza to an existing Docker Compose file. For information on creating a Docker Compose file, see the [official Compose docs][101].
+1. Replace `./config` with the path to the directory you created for the runner configuration.
+1. Run `docker compose up -d`.
+
+   You can safely ignore the error: `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
+
+[101]: https://docs.docker.com/compose/compose-application-model/
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="Workflow Automation" level="p" %}}
+1. Enter a name for your runner.
+1. Click **Workflows**.
+1. Create a directory on your host where the runner can store its configuration, such as `./config`. You can also use this directory to store any credentials required by the runner's connection.
+1. Deploy your runner with Docker or Docker Compose:
+{{< tabs >}}
+{{% tab "Docker" %}}
+1. Click **Docker**.
+1. Run the provided `docker run` command on your host, replacing `./config` with the path to the directory you created for the runner configuration.
+
+   You can safely ignore the error `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
+{{% /tab %}}
+
+{{% tab "Docker Compose" %}}
+1. Click **Docker Compose**.
+1. Create a `docker-compose.yaml` file and add the provided YAML, or add the `runner` stanza to an existing Docker Compose file. For information on creating a Docker Compose file, see the [official Compose docs][101].
+1. Replace `./config` with the path to the directory you created for the runner configuration.
+1. Run `docker compose up -d`.
+
+   You can safely ignore the error `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
+
+[101]: https://docs.docker.com/compose/compose-application-model/
+{{% /tab %}}
+{{< /tabs >}}
+
+{{% /collapse-content %}}
+
+When you see the **Ready to use** status, you can create a new connection for the runner or see it on the **Private Action Runners** page:
+- To create a new connection for the runner, click **Link Runner to New Connection** and select an integration.
+- Click **View Runner** to see the runner on the **Private Action Runners** page.
+
+See [Connect a runner](#connect-a-runner) for more information on pairing your runner with a connection.
+
+## Connect a runner
+
+Before you can use an action runner, you must pair it with one or more connections.
+
+To pair a runner to a connection:
+1. From the [Workflow Automation][8] or [App Builder][9] Connections page, click **New Connection**.
+1. Select the integration you want to connect with your private action runner. For a list of integrations that support private actions, see [Overview](#overview).
```

Comment:
Suggestion: see earlier comment about linking to a more specifically named subheading instead of Overview